### ISSUE TITLE
如何配置VoVNet-99 backbone？

### ISSUE THREAD

**Issue by kingofstu** [mengtan00/GeoBEV#1]
您好，我想使用VoVNet-99作为backbone在nusenes test datasets做测试，像论文中的table2那样，应该如何配置呢？ 祝好！

**Comment by mengtan00** [comment]
We follow the configuration of VoVNet-99 in [PETR](https://github.com/megvii-research/PETR). Besides, both training set val sets are used to train the model for evaluating on test set.

**Comment by kingofstu** [comment]
> We follow the configuration of VoVNet-99 in [PETR](https://github.com/megvii-research/PETR). Besides, both training set val sets are used to train the model for evaluating on test set.

Thanks for your reply! Can I understand your meaning as using  the following code:
'''
img_backbone=dict(
        type='VoVNetCP',
        spec_name='V-99-eSE',
        norm_eval=True,
        frozen_stages=-1,
        input_ch=3,
        out_features=('stage4','stage5',)),
    img_neck=dict(
        type='CPFPN',
        in_channels=[768, 1024],
        out_channels=256,
        num_outs=2), 
'''
to in place the code:
'''
img_backbone=dict(
        type='ResNet',
        depth=50,
        num_stages=4,
        out_indices=(1, 2, 3),
        frozen_stages=-1,
        norm_cfg=dict(type='BN', requires_grad=True),
        norm_eval=False,
        with_cp=True,
        style='pytorch'),
    img_neck=dict(
        type='CustomFPN',
        in_channels=[1024, 2048],
        out_channels=512,
        num_outs=1,
        start_level=0,
        out_ids=[0]),
'''
best wishes!

**Comment by mengtan00** [comment]
Yes. The out_channel of FPN should also be aligned and don't forget to load the pretrained weights.

**Comment by kingofstu** [comment]
> Yes. The out_channel of FPN should also be aligned and don't forget to load the pretrained weights.

Thanks a lot, I really appreciate your guidence!
